scio-test/src/test/scala/com/spotify/scio/values/SCollectionTest.scala

```diff
@@ -199,6 +199,11 @@ class SCollectionTest extends PipelineSpec {
       val p = sc.parallelize(Seq(Seq("a b", "c d"), Seq("e f", "g h"))).flatten
       p should containInAnyOrder(Seq("a b", "c d", "e f", "g h"))
     }
+
+    runWithContext { sc =>
```

Comment:
You can reuse the previous `runWithContext` like other tests and have `val p1 = /*Seq*/` and `val p2 = /*Option*/`.